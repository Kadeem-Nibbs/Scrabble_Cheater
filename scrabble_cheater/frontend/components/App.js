import React, { Component } from 'react'
import { Container, Grid } from 'semantic-ui-react'

import HeaderLeft from './Headers/HeaderLeft'
import BoardContainer from './Board/BoardContainer'
import ToggleGameTypeContainer from './ToggleGameType/ToggleGameTypeContainer'

import HeaderRight from './Headers/HeaderRight'
import RackContainer from './Rack/RackContainer'
import WordListContainer from './WordList/WordListContainer'
import UndoRedoClear from './UndoRedoClear/UndoRedoClear'

class App extends Component {
  render() {  
    return (
      <Container className="mt-50px">
        <Grid className="scrabble-container centered">
          <Grid.Column className="fixed-width-left-col">
            
            <HeaderLeft />
            <BoardContainer />

            <ToggleGameTypeContainer />
            <UndoRedoClear />

          </Grid.Column>
        <Grid.Column className="mt-34px fixed-width-right-col">

            <HeaderRight />
            <RackContainer />
            <WordListContainer />

        </Grid.Column>
        </Grid>
      </Container>
    )
  }
}

export default App