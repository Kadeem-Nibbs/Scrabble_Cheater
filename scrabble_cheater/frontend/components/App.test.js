import React from 'react'
import App from './App'

test('render App', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()    
});
