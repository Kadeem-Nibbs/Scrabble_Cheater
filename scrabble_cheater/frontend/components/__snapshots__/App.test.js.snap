// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App 1`] = `
<Container
  className="mt-50px"
>
  <Grid
    className="scrabble-container centered"
  >
    <GridColumn
      className="fixed-width-left-col"
    >
      <HeaderLeft />
      <Connect(BoardContainer) />
      <Connect(ToggleGameTypeContainer) />
      <Connect(UndoRedoClear) />
    </GridColumn>
    <GridColumn
      className="mt-34px fixed-width-right-col"
    >
      <HeaderRight />
      <Connect(RackContainer) />
      <Connect(WordListContainer) />
    </GridColumn>
  </Grid>
</Container>
`;
